app/Notifications/ServersSuspendedNotification.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
please do not use new notifications. we have a general notificion method which users can also overwrite, this one cannot be overwritten by users